plugins-compat-tester/src/main/resources/resultToReport.xsl

```diff
@@ -78,7 +78,7 @@
             <tr>
                 <xsl:for-each select="$testedJenkinsCoordinates" >
                     <th class="version">
-                        <xsl:value-of select="version" />
+                        <xsl:value-of select="version" />  / <xsl:value-of select="/report/testJdkVersion" />
```

Comment:
Could be on the next line?

Reply:
For now I didn't do that, but I wrapped both versions in a specific `span` and a class so we can style them separately. I might have a look later to see if/how we can make the columns more "responsive" (so we see versions more or less widely depending on the number of columns basically).